foreign_cc/private/framework.bzl

```diff
@@ -122,6 +127,11 @@ CC_EXTERNAL_RULE_ATTRIBUTES = {
         mandatory = False,
         default = [],
     ),
+    "local_defines": attr.string_list(
```

Comment:
Where is `local_defines` used?

Reply:
Same as `copts` - its used by the methods in `cc_common` to construct the flags to pass to the `CFLAGS` env variable when setting up the scripts. It allows a user to add a define e.g. `-DNDEBUG` for this target only.